README.md

```diff
@@ -339,3 +339,7 @@ Looking for a roadmap?
 
 - [Here is](https://github.com/atoum/atoum/labels/In%20progress) the work in progress,
 - [And there](https://github.com/atoum/atoum/milestones/*) what will come in next releases.
+
+## License
+
+atoum is released under the BSD-3-Clause License. See the bundled [LICENSE](LICENSE) file for details.
```

Comment:
Yes it should work in Github but to be sure it works with most markdown parsers, I prefer to keep this as is.